### PR TITLE
Create spring-boot.md

### PR DESCRIPTION
**Pull request by gomathyk**

### CODE REVIEW

File: en/docs/develop/spring-boot.md [hunk 1]
```diff
@@ -0,0 +1,399 @@
...
+                   .permitAll()
+                   .anyRequest()
+                   .authenticated()
+                   .and()
+                   .oauth2Login().loginPage("/oidc-login");
+       }
+    }
+    ```
+
+2. Have a Login Controller Class and render the login page when the browser is redirected to `/oauth-login`.
```
Comment (sherenewso2) [inline en/docs/develop/spring-boot.md:194]: Have a --> Create a?

please check this in other places as well

File: en/docs/develop/spring-boot.md [hunk 2]
```diff
@@ -0,0 +1,399 @@
...
+       </form>
+    </div>
+
+    ```
+
+### Customize the logout endpoint
+
+If you wish to customize the logout endpoint, follow the steps below. 
+
+1. Have a ConfigSecurity class by extending WebSecurityConfigurerAdapter and configure the logoutUrl
```
Comment (sherenewso2) [inline en/docs/develop/spring-boot.md:314]: monospace for class names

File: en/docs/develop/spring-boot.md [hunk 3]
```diff
@@ -0,0 +1,399 @@
...
+           oidcLogoutSuccessHandler.setPostLogoutRedirectUri(
+                   URI.create("http://localhost:8080/sprinb-boot-app"));
+
+           return oidcLogoutSuccessHandler;
+       }
+
+    }
+
+    ```
+####  Have a Logout button and redirect to custom logout (“/applogout”) url
```
Comment (sherenewso2) [inline en/docs/develop/spring-boot.md:350]: Have a --> Add ?

File: en/docs/develop/spring-boot.md [hunk 4]
```diff
@@ -0,0 +1,399 @@
...
+
+```html
+
+<div style="float:right">
+   <form method="post" th:action="@{/applogout}"  class="navbar-form navbar-right">
+       <button id="logout-button" type="submit" class="btn btn-danger">Logout</button>
+   </form>
+</div>
+
+```
```
Comment (sherenewso2) [inline en/docs/develop/spring-boot.md:364]: shall we add numbering for the steps?

File: en/docs/develop/spring-boot.md (hunk 3 above)
Comment (sherenewso2) [reply en/docs/develop/spring-boot.md:350]: Logout --> logout

File: en/docs/develop/spring-boot.md [hunk 5]
```diff
@@ -0,0 +1,399 @@
...
+
+## Read User Information
```
Comment (sherenewso2) [inline en/docs/develop/spring-boot.md:366]: subheadings in sentence case

File: en/docs/develop/spring-boot.md (hunk 4 above)
Comment (gomathyk) [reply en/docs/develop/spring-boot.md:364]: Have added steps if there are many. These have only one step and the sub sections are not in sequence so can't make them a list.

File: en/docs/develop/spring-boot.md [hunk 6]
```diff
@@ -0,0 +1,399 @@
...
+| --------------------- | ------------------------------      | -------------------------------------------                  |
+| Service Provider Name | your-application-name               | sample-app                                                   |
+| Description           | This is a spring-boot application   | This is a spring-boot application                            |
+| CallBack Url          | {baseUrl}/login/oauth2/code/wso2    | http://localhost:8080/sprinb-boot-app/login/oauth2/code/wso2 |
+| Logout Url            | {baseUrl}/login                     | http://localhost:8080/spring-boot-app                        |
+                        
+
+### Add the dependency 
+
+Add the following dependencies in the pom file in the spring-boot-project.
```
Comment (sherenewso2) [inline en/docs/develop/spring-boot.md:46]: in the pom file --> to the pom file

File: en/docs/develop/spring-boot.md [hunk 7]
```diff
@@ -0,0 +1,399 @@
...
+
+           return oidcLogoutSuccessHandler;
+       }
+    }
+    ```
+    
+2. Create a template at `src/main/resources/login.html` to display the logout option and have a logout button.
+
+
+3.  Add the `/logout` redirection when user clicks the Logout button.
```
Comment (sherenewso2) [inline en/docs/develop/spring-boot.md:299]: **Logout** button

**Review by sherenewso2 — APPROVED** [review]

**Pull request merged**